cmd/prombench/main.go

```diff
@@ -16,38 +16,53 @@ func main() {
 	app.HelpFlag.Short('h')
 
 	g := gke.New()
-	k8sGKE := app.Command("gke", `Google container engine provider - https://cloud.google.com/kubernetes-engine/.Requires`).
+	k8sGKE := app.Command("gke", `Google container engine provider - https://cloud.google.com/kubernetes-engine/`).
 		Action(g.NewGKEClient).
 		Action(g.ConfigParse)
-	k8sGKE.Flag("config", "yaml GKE config file used to create or delete the k8s cluster and nodes").
-		Default("../../config/cluster.yaml").
-		PlaceHolder("cluster.yaml").
-		Short('c').
-		ExistingFileVar(&g.ClusterConfigFile)
-	k8sGKE.Flag("auth", "json authentication file for the project - https://cloud.google.com/iam/docs/creating-managing-service-account-keys. If not set the tool will use the GOOGLE_APPLICATION_CREDENTIALS env variable (export GOOGLE_APPLICATION_CREDENTIALS=key.json)").
-		PlaceHolder("key.json").
+	k8sGKE.Flag("auth", "json authentication file for the project - https://cloud.google.com/iam/docs/creating-managing-service-account-keys. If not set the tool will use the GOOGLE_APPLICATION_CREDENTIALS env variable (export GOOGLE_APPLICATION_CREDENTIALS=service-account.json)").
+		Required().
+		PlaceHolder("service-account.json").
 		Short('a').
 		ExistingFileVar(&g.AuthFile)
+	k8sGKE.Flag("config", "GKE cluster-config yaml file").
+		PlaceHolder("cluster.yaml").
+		Short('c').
+		Default("config/cluster.yaml").
+		ExistingFileVar(&g.ConfigFile)
+	k8sGKE.Flag("pool", "GKE Nodepool-config yaml file").
```

Comment:
I would say lets keep both configs in `cluster.yaml` keeps it less confusing and no need for a separate flag.

Reply:
I kept it separate because when benchmarking is triggered, just prombench node-pools should be made, not prow. Since [createClusterRequest](cloud.google.com/go/container/apiv1) & [createNodePoolRequest](https://godoc.org/cloud.google.com/go/container/apiv1#ClusterManagerClient.CreateNodePool) are different structs, I have made 2 different files.

Reply:
my idea is to use the same file, but use the parts that a needed for the command that is executed.

Reply:
so you will parse the entyre file, but just use the `nodepools` for `nodePool create` and the `cluster` for `cluster create`